readme.md

```diff
@@ -46,7 +61,7 @@ Number of columns to wrap the text to.
 
 - [slice-ansi](https://github.com/chalk/slice-ansi) - Slice a string with ANSI escape codes
 - [chalk](https://github.com/chalk/chalk) - Terminal string styling done right
-
+- [jsesc](https://github.com/mathiasbynens/jsesc) - Generate ascii only output from unicode strings. This is useful for creating testing fixtures.
```

Comment:
Happy to remove it. I added it because writing fixtures for tests is a pain in the butt without using a tool like `jsesc`.